dali/imgcodec/decoders/libtiff/tiff_libtiff.cc

```diff
@@ -236,19 +246,33 @@ DecodeResult LibTiffDecoderInstance::Decode(DecodeContext ctx,
   if (info.is_palette)
     return {false, make_exception_ptr(std::logic_error("Palette images are not yet supported"))};
 
-  // TODO(skarpinski) Support tiled images
-  if (info.is_tiled)
-    return {false, make_exception_ptr(std::logic_error("Tiled images are not yet supported"))};
+  if (info.is_tiled && info.tile_width % 16 != 0 && info.tile_height % 16 != 0) {
```

Comment:
```suggestion
  if (info.is_tiled && (info.tile_width % 16 != 0 || info.tile_height % 16 != 0)) {
```
Shouldn't be both a multiple of 16?